readme.md

```diff
@@ -32,37 +32,37 @@ brew install --cask sequel-ace
 ## Building Locally
 
 To run Sequel Ace locally from XCode, please:
-- download `.zip` archive of this repo / clone locally
+- download `.zip` archive of this repo/clone locally
 - open `sequel-ace.xcworkspace` and run `Sequel Ace Local Testing` schema
 
-If you encounter any issue, let us know by [creating a new issue](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose).
+If you encounter any issues, let us know by [creating a new issue](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose).
 
 ## Moving Saved Connections from Sequel Pro
 
-To move your favorites, please check [migration guide](https://sequel-ace.com/get-started/migrating-from-sequel-pro.html).
+To move your favourites, please check the [migration guide](https://sequel-ace.com/get-started/migrating-from-sequel-pro.html).
```

Comment:
This change here is the only one I don't like - "favourites" is the British English form but perhaps we should stick with the original US English form of "favorites"